cache.h

```diff
@@ -918,13 +920,18 @@ extern char *git_replace_ref_base;
 
 extern int fsync_object_files;
 extern int core_preload_index;
-extern int core_apply_sparse_checkout;
 extern const char *core_virtualfilesystem;
 extern int core_gvfs;
 extern int precomposed_unicode;
 extern int protect_hfs;
 extern int protect_ntfs;
 extern const char *core_fsmonitor;
+extern int core_use_gvfs_helper;
+extern const char *gvfs_cache_server_url;
+extern struct strbuf gvfs_shared_cache_pathname;
+
+int core_apply_sparse_checkout;
```

Comment:
can't tell from the web-view context, so i'll just make a note here.

should these be `extern` ?

Reply:
`core_apply_sparse_checkout` should indeed be `extern` here.